rdmd/verbose.d

```diff
@@ -0,0 +1,13 @@
+/**
+Provides functionality for verbose rdmd output
+*/
+module rdmd.verbose;
+
+void yap(size_t line = __LINE__, T...)(auto ref T stuff)
+{
+    import rdmd.args : RDMDGlobalArgs;
+    import std.stdio : stderr;
+    if (!RDMDGlobalArgs.chatty) return;
+    debug stderr.writeln(line, ": ", stuff);
+    else stderr.writeln(stuff);
+}
```

Comment:
This module is currently disproportionately small. Are you planning to expand it?

Reply:
There are 2 reasons why that module is being split out.

First, it's needed in order to modularize out any other functionality that uses `yap`.  For example, `rdmd.filesystem` needs it, and other modules likely will in future.  Ideally in the longer run I will also encapsulate there any global state required for dry-run/verbose output to work.

Second, I do think there's scope in future for a rewrite of how verbose output is handled by rdmd.  Right now there is not really a good separation of concerns between dry-run, versus verbose, versus debugging aids.  Having a dedicated module will make that kind of work easier.

Reply:
All right. `rdmd.main` still imports this, but all remaining uses of `yap` there look like good candidates to be encapsulated and moved away from the main application logic.